### ISSUE TITLE
Is there a way to make SSLClient work without rand_pin?

### ISSUE THREAD

**Issue by mikeme2009** [OPEnSLab-OSU/SSLClient#62]
I designed a PCB with ESP32 but the only pins free are 17, 21, 25, 27 and I receive  in platformio  
[E][esp32-hal-adc.c:123] __analogRead(): Pin 21 is not ADC pin!
Is there a way to make SSLClient work without rand_pin?
Thanks

**Comment by HebertWP** [comment]
Yep, it's only part inside  of the library that i really understood, it's make a analogic read in this pin then give the value the library core with is a encryption librqrar written in C, so make the alteration  inside the library to call esp random  generator

**Comment by mikeme2009** [comment]
Thank you

**Comment by mikeme2009** [comment]
Hi again,

I managed to communicate with the server, but Serial.write (buffer, len); print the entire message, including header and body. How can I print only the body?  Something like http.getString (); from the HTTPClient.h library
